robottelo/ui/gpgkey.py

```diff
@@ -154,28 +140,33 @@ def assert_key_from_product(self, name, product, repo=None):
                 self.wait_for_ajax()
                 self.wait_until_element(locators["repo.gpg_key_edit"]).click()
                 self.wait_for_ajax()
-                element = Select(self.find_element
-                                 (locators["repo.gpg_key_update"]
-                                  )).first_selected_option.text
+                element = Select(
+                    self.find_element(locators["repo.gpg_key_update"])
+                ).first_selected_option.text
                 if element == '':
                     return None
                 else:
-                    raise Exception(
-                        "GPGKey '%s' is still assoc with selected repo" % name)
+                    raise UIError(
+                        'GPGKey "{0}" is still assoc with selected repo'.
+                        format(name)
```

Comment:
Minor thing, but can you place the dot operator on this line? Also, `s/assoc/associated/` if it'll fit.

    'GPGKey "{0}" is still associated with selected repo'
    .format(name)